pkg/collector/metric/node_metric.go

```diff
@@ -106,7 +123,7 @@ func getCPUArchitecture() (string, error) {
 		if err := dec.Decode(&p); err == io.EOF {
 			break
 		}
-		if strings.HasPrefix(myCPUModel, p.Name) {
+		if strings.Contains(myCPUModel, p.Architecture) {
```

Comment:
if we can get uniform cpu architecture names, we don't need to read `normalized_cpu_arch.csv` any more; the file is there to help resolve different naming conventions (temporary workaround)